### ISSUE TITLE
crane-scheduler-controller 获取prometheus指标失败

### ISSUE THREAD

**Issue by tommylxf** [gocrane/crane-scheduler#8]
crane-scheduler-controller版本：0.0.23
craned版本：0.5.0
k8s版本：1.21.10
docker版本：19.3.14
系统版本：Ubuntu 20.04.3 LTS

手动调用prometheus api接口可以获取到对应的指标
curl -g http://prometheus-k8s.monitoring.svc.cluster.local:9090/api/v1/query?query=cpu_usage_avg_5m
{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"cpu_usage_avg_5m","instance":"ceph-01"},"value":[1656488784.456,"2.7104166665715894"]},{"metric":{"__name__":"cpu_usage_avg_5m","instance":"ceph-02"},"value":[1656488784.456,"1.9583333333351618"]},{"metric":{"__name__":"cpu_usage_avg_5m","instance":"ceph-03"},"value":[1656488784.456,"2.6000000000931323"]},{"metric":{"__name__":"cpu_usage_avg_5m","instance":"node-01"},"value":[1656488784.456,"4.0291666666841195"]},{"metric":{"__name__":"cpu_usage_avg_5m","instance":"node-04"},"value":[1656488784.456,"6.870833333426461"]},{"metric":{"__name__":"cpu_usage_avg_5m","instance":"ykj"},"value":[1656488784.456,"5.891666666672492"]}]}}/ # 

curl -g http://prometheus-k8s.monitoring.svc.cluster.local:9090/api/v1/query?query=mem_usage_avg_5m
{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"mem_usage_avg_5m","instance":"ceph-01","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-sn9lp"},"value":[1656488826.549,"32.75862684328356"]},{"metric":{"__name__":"mem_usage_avg_5m","instance":"ceph-02","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-dgd54"},"value":[1656488826.549,"15.044355868789062"]},{"metric":{"__name__":"mem_usage_avg_5m","instance":"ceph-03","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-td7k2"},"value":[1656488826.549,"34.21244570563606"]},{"metric":{"__name__":"mem_usage_avg_5m","instance":"node-01","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-zzxmd"},"value":[1656488826.549,"57.21168005976536"]},{"metric":{"__name__":"mem_usage_avg_5m","instance":"node-04","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-2zkgk"},"value":[1656488826.549,"72.4792896090607"]},{"metric":{"__name__":"mem_usage_avg_5m","instance":"ykj","job":"node-exporter","namespace":"monitoring","pod":"node-exporter-xfq4n"},"value":[1656488826./

**Comment by autumn0207** [comment]
@tommylxf 

Currently scheduler-controller only supports pulling data by `node internal IP` instead of `node name`, so you need to modify the configuration of Prometheus service discovery for node-exporter  to make `node internal IP` or ` node internal IP : port`  as the value of label key `instance` .

For example:
```yaml
    - job_name: kubernetes-node-exporter
      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
        insecure_skip_verify: true
      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
      scheme: https
      kubernetes_sd_configs:
      ...
      # delete the following two lines
      # Host name
     - source_labels: [__meta_kubernetes_node_name]
       target_label: instance
      ...
```

**Comment by xieydd** [comment]
fixed already in #19
